### ISSUE TITLE
Number of epochs

### ISSUE THREAD

**Issue by sehsanh** [yongxuUSTC/DNN-for-speech-enhancement#20]
Hi Dr.Xu,
I am writing your code in python and using random initializing for layers. I also read your papers, but I couldn't find the number of epochs and learning rate that you use for training on each 100 hours of noisy data (from 6 SNRs and 104 types of noises). What is the best choices for these parameters?
Thanks for your help
